src/src/PluginStructs/P104_data_struct.cpp

```diff
@@ -1154,8 +1154,9 @@ void reverseStr(String& str) {
     uint16_t n = str.length();
  
     // Swap character starting from two corners
-    for (uint16_t i = 0; i < n / 2; i++)
-        std::swap(str[i], str[n - i - 1]);
+    for (uint16_t i = 0; i < n / 2; i++) {
+      std::swap(str[i], str[n - i - 1]);
+    }
```

Comment:
There is a reverse() function in String class.

Reply:
This was (number) one of 5 options I found on a c++ site, the cleanest, simplest, and also the only solution presented that does the swap 'in-place', that's why I ~~stole~~ used it. For odd length strings the center character isn't touched, AFAICS.